src/unittest/test.cpp

```diff
@@ -1349,7 +1350,7 @@ Markdown lol \\|\\|spoiler\\|\\| \\~\\~strikethrough\\~\\~ \\`small \\*code\\* b
 
 		bool message_edit_tested = false;
 		bot.on_message_update([&](const dpp::message_update_t &event) {
-			if (event.msg.author == bot.me.id) {
+			if (event.msg.author.id == bot.me.id) {
```

Comment:
What's the difference here? The parent class of author handles this right?

Reply:
It stops an implicit conversion to managed, which is now basically an abstract base class.

Reply:
Ah right, I get you!